### ISSUE TITLE
Headings should be in ordinal order to make sure they are exposed correctly to screen readers.

### ISSUE THREAD

**Issue by ciaoaaron** [BluKirb/SoulCall-Website#2]
https://github.com/BluKirb/SoulCall-Website/blob/19dc08da49431bfd5028320bc2585ad94e37171d/index.html#L19

This should be H1
